src/transformers/models/chameleon/processing_chameleon.py

```diff
@@ -159,6 +159,21 @@ def decode(self, *args, **kwargs):
         """
         return self.tokenizer.decode(*args, **kwargs)
 
+    # Copied from transformers.models.blip.processing_blip.BlipProcessor.post_process_image_text_to_text
+    def post_process_image_text_to_text(self, generated_outputs):
```

Comment:
Agreed that it's repetitive, but since `ProcessorMixin` is also used by models that are not vlms, wouldn't it be a problem to put this function there?